src/core/Form/SearchInput/SearchInput.md

```diff
@@ -171,6 +174,64 @@ import { SearchInput } from 'suomifi-ui-components';
 />;
 ```
 
+### Search suggestions
+
+You can provide search suggestions for the user using the `autosuggest` and `suggestions` props. The suggestions are shown in a popover list under the input field. Provide a descriptive `suggestionHintText` to let screen reader users know there will be a suggestion list under the component, as the usual accessible pattern is not available for search input component. Also provide `ariaOptionsAvailableText` to inform screen reader users about the updating amount of suggestions.
+
+When the user selects a suggestion from the list, the `onSuggestionSelected` callback gets called with the `uniqueId` of the element.
+
+It's recommended to use debounce on fetching the suggestions to avoid unnecessary fetches. Do not use debounce prop of the component for this purpose.
```

Comment:
I think this needs a bit more clarification for the user. Why should they not use the native debounce? What should they do instead? (Make them pay attention to the example below)